lib/raven/processor/sanitizedata.rb

```diff
@@ -5,6 +5,7 @@ class Processor::SanitizeData < Processor
     INT_MASK = 0
     DEFAULT_FIELDS = %w(authorization password passwd secret ssn social(.*)?sec)
     CREDIT_CARD_RE = /^(?:\d[ -]*?){13,16}$/
+    REGEX_SPECIAL_CHARACTERS = %w(. $ ^ { [ ( | ) * + ?)
```

Comment:
ah, I see, you can't keep backwards compat with my suggestion. You'd have to specify them as `/\Aname\Z/` instead of `"name"`. Kinda weird.

So my recommendation is probably:

1. Merge this as is, release in next point release.
2. Next major version, switch to doing exact matching of strings and allowing regexes. It's backwards incompatible, but probably less surprising.